### ISSUE TITLE
Limit amount messages displayed and add ability to "load more"

### ISSUE THREAD

**Issue by colinmacdonald** [goinstant/chat#26]
For performance reasons we should limit the amount of messages that will appear in the chat widget, if you scroll to the top you should be able to "load more" messages. 

The maximum number of messages to display could be an option.

**Comment by byosko** [comment]
@colinmacdonald there's already a setting for deleting messages after X time, I believe it's 24 hours, but I guess this would just be about message display right and not storing messages?

**Comment by colinmacdonald** [comment]
Yeah, this would allow all the messages to be stored and just loaded as needed. It's mainly for performance at initialization. If you have say 1000 messages and you are switching pages and re-initializing the widget often you need to load 1000 messages into the view when maybe only the last 100 are relevant to whats currently going on on the page.

This especially useful for for IE8, IE8 rendering even 100 messages into the view is pretty slow (talking seconds to initialize the widget).

**Comment by byosko** [comment]
Totally makes sense. Deleting messages was really a stop gap for the demo in our docs, but it could also be useful for some people. As long as the differences are clear.

**Comment by colinmacdonald** [comment]
A cool way of doing this would be to automatically load more messages as you scroll near the top, something similar to facebook's news feed.
